admin-dev/themes/new-theme/js/pages/product/product-map.js

```diff
@@ -103,6 +103,7 @@ export default {
     editionForm: 'form[name="combination_form"]',
     editCombinationButtons: '.edit-combination-item',
     tableRow: {
+      combinationImg: '.combination-image',
```

Comment:
Maybe you should used an "indexed" selector like the other ones, jjst to be consistent?